charge_changes.json

```diff
@@ -0,0 +1,12 @@
+{"dcd_save_freq": 100,
```

Comment:
do you want to commit this file?

Reply:
No, I do not think it is necessary, at least not now